org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DebugAdapterContext.java

```diff
@@ -123,4 +151,45 @@ public String getSourceUri(int sourceReference) {
     public int createSourceReference(String uri) {
         return this.sourceReferences.create(uri);
     }
+    
+    @Override
+    public List<ThreadReference> safeGetAllThreads() {

```

Comment:
In PR#69, i already move safeGetAllThreads() to DebugUtility. No need to put it in DebugAdapterContext.